### DISCUSSION TITLE
cordova to capacitor migration

### DISCUSSION THREAD

**Discussion by kiranharidas187** [Sunbird-Ed/Community#13] [Feature request]
Looks like Ionic is slowly moving away from Cordova. In the latest release of ionic, the creation of a new project sets capacitor as default.
Do we have any plan to migrate the SunbirdEd-mobile app from Cordova to the capacitor in near future?

**Comment by sharathkashyap** [comment]
@kiranharidas187 Acknowleged This will be added into the roadmap.
cc: @luckynvdu

**Comment by luckynvdu** [comment]
@kiranharidas187 Will will be picking up this as part of this ticket https://project-sunbird.atlassian.net/browse/SB-30463 and will be released in 5.0 major release

@sharathkashyap

**Reply by kiranharidas187** [reply]
Didn't understand @luckynvdu. You mean the story is being worked upon and will be released as part of 5.0?

**Reply by kiranharidas187** [reply]
@luckynvdu 
One doubt from my side, so in release-4.10.2 we will be doing complete regression of the app right?

cc - @vijiurs

**Reply by luckynvdu** [reply]
@kiranharidas187 I meant this story SB-30463 will be available as part of 5.0 release which is a major release.
As 4.10.2 is an intermediate release hence there will be only functional testing and no regression testing

**Comment by sharathkashyap** [comment]
@kiranharidas187 We are definitely evaluating this migration as a Spike first inside the Sprint. Since SunbirdED Mobile App has lot of customised plugins. This would require us to test the migration of plugins into capacitor framework first. It also needs to be evaluated in terms of code change that would get introduced since we will be retiring cordova. This might be a big change considering the releases and requirements.

We will get back to forum with the spike results.
